server/spec/cards.spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Post request /api/v1/cards", () => {
+  const request = require("request")
+  const base_url = "http://localhost:5000/api/v1/cards"
+
+  it("returns 201 code when card is created", (done) => {
+    request({
+      url: base_url,
+      method: "POST",
+      json: true,
+      body: { description: "Test"},
+    }, (error, response, body) => {
+      expect(response.statusCode).toEqual(201)
+      done()
+    });
+  });
+});
```

Comment:
@nikitasakov thoughts about adding a test for checking the negative result? 

`400` code with error message